web/src/app/common/InvalidUser/InvalidUserComponent.js

```diff
@@ -0,0 +1,22 @@
+import {
+  PageHeader
+} from 'react-bootstrap';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { LinkContainer } from 'react-router-bootstrap';
+import styles from './InvalidUserStyles.scss';
+
+class InvalidUserComponent extends Component {
```

Comment:
Good point. I think we should keep this (for now) as is. As we proceed and it becomes apparent we should share a component for missing resources, it will not be hard to add.